Week-7/q6.js

```diff
@@ -0,0 +1,38 @@
+var MyQueue = function () {
```

Comment:
this can be declared in common file maybe as it is used in multiple places also use const